### ISSUE TITLE
reintroduce jaeger links to web dashboard

### ISSUE THREAD

**Issue by adleong** [linkerd/linkerd2#5329]
The linkerd-viz extension should support a `jaegerAddr` value which configures the address of a jaeger instance.  If set, the dashboard should display a link to the jaeger dashboard for each workload.  This is similar to how it already displays a link to the grafana dashboard for each resource.

**Comment by dadjeibaah** [comment]
@adleong, I tested this on the main branch, it seems like this feature is already supported.  It's possible that we may need to update docs so that we clearly describe how to get this working. 

I built linkerd and first installed the jaeger extension. After all checks passed with `bin/linkerd jaeger check`,  I then installed `linkerd-viz` with this command. `bin/linkerd viz install --set jaegerUrl=jaeger.linkerd-jaeger.svc.cluster.local:16686 | kaf - `    This sets the jaeger URL in the web container manifest to the service URL provided by linkerd jaeger service. After the `viz` install I was able to see a column in the resource detail table with jaeger links. Clicking on a link took me to the jaeger trace page for the chosen service.

One other option we could do is add a new flag to `linkerd viz install` that takes in a url that points to the dashboard. That would mean that the viz install command would look slightly different from the jaeger command.
